### PR TITLE
Complete all tasks on task list

### PR DESCRIPTION
**Pull request by lanej0**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Adds an e2e test to check that the task list is hidden when all tasks are completed.  Because of how this is implemented, it probably shouldn't be an e2e test and should be a lower-level test.  It didn't take long to write, so if we want to reject this no hard feelings.

Closes https://github.com/woocommerce/woocommerce-quality/issues/593

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. `pnpm test:e2e activate-and-setup`
2. Ensure CI passes

### CODE REVIEW

**Comment by rodelgc** [comment]
Hi @lanej0 Seems this PR is out of date with trunk. If you could update it, that would be great. I could take a look after.

**Comment by lanej0** [comment]
> Hi @lanej0 Seems this PR is out of date with trunk. If you could update it, that would be great. I could take a look after.

I updated the branch, but it looks like there have already been other changes merged.  Unless there's a merge conflict, it's probably okay to look at these PRs as is (they're going to constantly be out of date with the base branch)

File: plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/task-list-tasks.spec.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+					console.log( 'Option deleted successfully.' );
+					resolve( stdout );
+				} );
+			} );
+		} );
+
+		test( 'Task list is not displayed when all tasks complete', async ( {
+			page,
+		} ) => {
+			await page.goto( '/wp-admin/admin.php?page=wc-admin' );
```
Comment (Stojdza) [inline plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/task-list-tasks.spec.js:61]: Following changes from https://github.com/woocommerce/woocommerce/pull/53228 and supporting multisite setup, URLs should start without `/` or with `./`

**Comment by adimoldovan** [comment]
> Because of how this is implemented, it probably shouldn't be an e2e test and should be a lower-level test. It didn't take long to write, so if we want to reject this no hard feelings.

+1 on this. Not an e2e for sure, and I would not add it to the already bloated suite TBH.

**Comment by lanej0** [comment]
> +1 on this. Not an e2e for sure, and I would not add it to the already bloated suite TBH.

Okay, I closed the issue as "not planned" and I'm going to close this PR.  Thanks for the gut check!

**Pull request closed without merging**